### ISSUE TITLE
Default log output should include Sb account name, topic, and subscription

### ISSUE THREAD

**Issue by jcookems** [azure-contrib/socket.io-servicebus#27]
When looking at the log output of the chat app, I cannot tell which subscription it is using. Given the extraordinary amount of other output in the console log, I'd expect that basic info to be included:

```
C:\dd\git\jcooke\socket.io-servicebus\examples\chat>node app.js
The "sys" module is now called "util". It should have a similar interface.
   info  - socket.io started
SbStore has new subscriber for handshake events
SbStore has new subscriber for connect events
SbStore has new subscriber for open events
SbStore has new subscriber for join events
SbStore has new subscriber for leave events
SbStore has new subscriber for close events
SbStore has new subscriber for dispatch events
SbStore has new subscriber for disconnect events
   app listening on http://0.0.0.0:30001
   debug - client authorized
   info  - handshake authorized AMAVV73v9NwloaeteQAX
   debug - setting request GET /socket.io/1/xhr-polling/AMAVV73v9NwloaeteQAX?t=1361999102315
SbStore has new subscriber for heartbeat-clear:AMAVV73v9NwloaeteQAX events
SbStore has new subscriber for disconnect-force:AMAVV73v9NwloaeteQAX events
SbStore has new subscriber for dispatch:AMAVV73v9NwloaeteQAX events
   debug - setting poll timeout
   debug - client authorized for
   debug - clearing poll timeout
   debug - xhr-polling writing 1::
   debug - set close timeout for client AMAVV73v9NwloaeteQAX
SbStore unsubscribing from disconnect-force:AMAVV73v9NwloaeteQAX
SbStore unsubscribing from heartbeat-clear:AMAVV73v9NwloaeteQAX
SbStore unsubscribing from dispatch:AMAVV73v9NwloaeteQAX
SbStore has new subscriber for dispatch:AMAVV73v9NwloaeteQAX events
SbStore has new subscriber for message:AMAVV73v9NwloaeteQAX events
SbStore has new subscriber for disconnect:AMAVV73v9NwloaeteQAX events
   debug - setting request GET /socket.io/1/xhr-polling/AMAVV73v9NwloaeteQAX?t=1361999102343
SbStore has new subscriber for heartbeat-clear:AMAVV73v9NwloaeteQAX events
SbStore has new subscriber for disconnect-force:AMAVV73v9NwloaeteQAX events
SbStore has new subscriber for dispatch:AMAVV73v9NwloaeteQAX events
   debug - setting poll timeout
SbStore unsubscribing from dispatch:AMAVV73v9NwloaeteQAX
   debug - discarding transport
   debug - cleared close timeout for client AMAVV73v9NwloaeteQAX
SbStore unsubscribing from disconnect-force:AMAVV73v9NwloaeteQAX
SbStore unsubscribing from heartbeat-clear:AMAVV73v9NwloaeteQAX
SbStore unsubscribing from dispatch:AMAVV73v9NwloaeteQAX
...
```

**Comment by christav** [comment]
I think this is fixed as part of addressing #13

**Comment by christav** [comment]
Closing, fixed.
